ompi/mca/pml/base/pml_base_frame.c

```diff
@@ -214,7 +216,6 @@ static int mca_pml_base_open(mca_base_open_flag_t flags)
         if( (NULL == default_pml || NULL == default_pml[0] ||
              0 == strlen(default_pml[0])) || (default_pml[0][0] == '^') ) {
             opal_pointer_array_add(&mca_pml_base_pml, strdup("ob1"));
-            opal_pointer_array_add(&mca_pml_base_pml, strdup("yalla"));
```

Comment:
This is definitely a correct fix, but it begs the question: why on earth do we have a hard-coded list of PMLs here?

@hjelmn Don't we have a list of autogen/configure-found PMLs somewhere that we can use instead?